src/mgmt/deploy-dashboard-vm.py

```diff
@@ -339,26 +341,33 @@ def create_kickstart(ks_filename, cfg_filename):
                 ks.write("echo network --activate --onboot=true --noipv6"
                          " --device='{}' --bootproto=static --ip='{}'"
                          " --netmask='{}' --hostname='{}'"
-                         " --gateway='{}' --nameserver='{}' >> {}\n".
+                         " --gateway='{}' --nameserver='{}' --mtu='{}'>> {}\n".
                          format(
                              tokens[0],
                              tokens[1],
                              tokens[2],
                              hostname,
                              gateway,
                              nameservers,
+                             tokens[3],
                              ks_include))
+                ks.write("echo eth0='{}' >> {}\n".
```

Comment:
Can we change the variable name to something like eth0_mtu or something like that?

Reply:
will change it to eth0_mtu